Commands/Public/contacts.js

```diff
@@ -109,9 +109,9 @@ module.exports = async(client, msg, suffix) => {
 				omsg = await omsg.edit("", { embed: {
 					color: config.colors.contacts,
 					title: `Add a name for ${number.id}`,
-					description: "Please enter a name for the number. (max 20 characters)",
+					description: "Please enter a name for the number.",
 					footer: {
-						text: "Press (0) to hangup. This call will automatically be hung up after 1 minutes of inactivity.",
+						text: "Max 20 characters. (0) to hangup. This call will automatically be hung up after 1 minute of inactivity.",
```

Comment:
This isn't going to work. People don't even read the title.
If you're reworking this anyway, make it a feedback loop.

Reply:
why are users so bloody dumb